src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -48,8 +49,11 @@ def reinit_data(self):
             # Example output: ASIC_STATE:SAI_OBJECT_TYPE_BRIDGE_PORT:oid:0x3a000000000616
             bridge_port_id = s[45:]
             ent = self.db_conn.get_all(mibs.ASIC_DB, s, blocking=True)
-            port_id = ent[b"SAI_BRIDGE_PORT_ATTR_PORT_ID"][6:]
-            self.if_bpid_map[bridge_port_id] = port_id
+            try:
+                port_id = ent[b"SAI_BRIDGE_PORT_ATTR_PORT_ID"][6:]
```

Comment:
>SAI_BRIDGE_PORT_ATTR_PORT_ID [](start = 32, length = 28)

suggest check dict keys instead of try-catch.

Reply:
`if key in dict` goes when it is a condition that we needs to deal with. since this is an abnormal situation, i think it fits better in `try/catch` scenario.